pkg/grpc/databroker/changeset.go

```diff
@@ -37,6 +38,7 @@ func (cs *changeSet) Remove(typ string, id string) {
 		Type:      typ,
 		Id:        id,
 		DeletedAt: cs.now,
+		Data:      &anypb.Any{TypeUrl: typ},
```

Comment:
I'm not sure what this will do. The data would be nil. It may fail in marshaling and/or unmarshaling.

Reply:
Apparently it's fine 🤷 .